Exericise2.py

```diff
@@ -0,0 +1,6 @@
+#Write a Python program to find those numbers which are divisible by 7 and multiple of 5, between 1500 and 2700 (both included).
+z = []
+for a in range(1499, 2701):
```

Comment:
for numbers between 1500 and 2700, I think your range should start from `1500` instead of `1499`